asynchronous1.js

```diff
@@ -0,0 +1,15 @@
+1. const MyPromise = class {
+    constructor (callback) {this.promise = new Promise (callback)}
```

Comment:
Code looks much better with indentations inside brackets
`{
     this.promise = new Promise (callback);
}`